tests/main_test.rs

```diff
@@ -794,6 +794,49 @@ mod tests {
         assert!(result.is_ok());
     }
 
+    #[test]
+    fn encodes_and_decodes_an_ec_token_using_jwks_secret() {
+        let body: String = "{\"field\":\"value\"}".to_string();
+        let encode_matcher = App::command()
+            .try_get_matches_from(vec![
+                "jwt",
+                "encode",
+                "-A",
+                "ES256",
+                "--kid",
+                "4h7wt2IHHu_RLR6OtlZjCe_mIt8xAReS0cDEwwWAeKU",
+                "--exp",
+                "-S",
+                "@./tests/private_ecdsa_key.pk8",
+                &body,
+            ])
+            .unwrap();
+        let encode_matches = encode_matcher.subcommand_matches("encode").unwrap();
+        let encode_arguments = EncodeArgs::from_arg_matches(encode_matches).unwrap();
+        let encoded_token = encode_token(&encode_arguments).unwrap();
+        // "$(cat ./tests/pub_ecdsa_jwks.json)"
+        let jwks = r#"{"keys":[{"use":"sig","kty":"EC","kid":"4h7wt2IHHu_RLR6OtlZjCe_mIt8xAReS0cDEwwWAeKU","crv":"P-256","x":"w7JAoU_gJbZJvV-zCOvU9yFJq0FNC_edCMRM78P8eQQ","y":"wQg1EytcsEmGrM70Gb53oluoDbVhCZ3Uq3hHMslHVb4"},{"use":"enc","kty":"EC","kid":"4h7wt2IHHu_RLR6OtlZjCe_mIt8xAReS0cDEwwWAeKU","crv":"P-256","x":"w7JAoU_gJbZJvV-zCOvU9yFJq0FNC_edCMRM78P8eQQ","y":"wQg1EytcsEmGrM70Gb53oluoDbVhCZ3Uq3hHMslHVb4"}]}
+"#;
```

Comment:
Shouldn't this use the `@` file notation rather than a raw string, similar to the encode step above?

Reply:
The idea is that since typically the JWKS is hosted on a remote server, that needs to be download over https, this allows to validate a JWT signature as a one-liner: 

for example: 

```
jwt decode $JWT_TOKEN --secret "$( curl -s https://www.googleapis.com/oauth2/v3/certs)"  --alg RS256
```

Instead of 
```
## first download JWKS as JSON file
curl -s https://www.googleapis.com/oauth2/v3/certs -o /tmp/jwks.json
## then validate the JWT signature
jwt decode $JWT_TOKEN --secret @/tmp/jwks.json  --alg RS256
```


Also there is another test that uses the `@./tests/pub_ecdsa_jwks.json` file notation here https://github.com/mike-engel/jwt-cli/blob/main/tests/main_test.rs#L782.